src/main/java/org/takes/rq/RqMultipart.java

```diff
@@ -506,6 +491,61 @@ private static StringBuilder fakeBody(final Request... dispositions)
                 .append(String.format("--%s--", BOUNDARY));
             return builder;
         }
+
+        /**
+         * A private inner class for the specific implementation
```

Comment:
@sebing please remove `A private inner class` - this is obvious from the code - what else could be `private static class`?